cfme/fixtures/pytest_selenium.py

```diff
@@ -982,7 +982,8 @@ def all_selected_options(self):
         Selenium's all_selected_options iterates over ALL of the options, this directly returns
         only those that are selected.
         """
-        return execute_script("return arguments[0].selectedOptions;", element(self))
+        return execute_script(
+            "return arguments[0] ? arguments[0].selectedOptions || [] : [];", element(self))
```

Comment:
So, call me nuts or something, but I'm not seeing why this is necessary. This only happens if we pass something that isn't a select element to the JS, which should (or should be made to) never happen. Anything that is a select element or otherwise implements [HTMLSelectElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement) will have a 'selectedOptions' attr.